trustpinning/certs.go

```diff
@@ -86,7 +86,7 @@ We shall call this: TOFUS.
 
 Validation failure at any step will result in an ErrValidationFailed error.
```

Comment:
Apologies it's collapsed, but I think there was a reference to cert stores above in this comment :)